sources/procfs.go

```diff
@@ -125,68 +125,68 @@ func newLustreProcMetric(filename string, promName string, source string, path s
 func (s *lustreSource) generateOSTMetricTemplates() error {
 	metricMap := map[string][]lustreHelpStruct{
 		"obdfilter/*": {
-			{"blocksize", "blocksize", "Filesystem block size in bytes"},
-			{"brw_size", "brw_size", "Block read/write size in bytes"},
-			{"brw_stats", "pages_per_bulk_rw", pagesPerBlockRWHelp},
-			{"brw_stats", "discontiguous_pages", discontiguousPagesHelp},
-			{"brw_stats", "disk_ios_in_flight", diskIOsInFlightHelp},
-			{"brw_stats", "io_time", ioTimeHelp},
-			{"brw_stats", "disk_io_size", diskIOSizeHelp},
-			{"degraded", "degraded", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded"},
-			{"filesfree", "filesfree", "The number of inodes (objects) available"},
-			{"filestotal", "filestotal", "The maximum number of inodes (objects) the filesystem can hold"},
-			{"grant_compat_disable", "grant_compat_disable", "Binary indicator as to whether clients with OBD_CONNECT_GRANT_PARAM setting will be granted space"},
-			{"grant_precreate", "grant_precreate", "Maximum space in bytes that clients can preallocate for objects"},
-			{"job_cleanup_interval", "job_cleanup_interval", "Interval in seconds between cleanup of tuning statistics"},
-			{"job_stats", "job_read_samples", readSamplesHelp},
-			{"job_stats", "job_read_minimum", readMinimumHelp},
-			{"job_stats", "job_read_maximum", readMaximumHelp},
+			{"blocksize", "blocksize_bytes", "Filesystem block size in bytes"},
+			{"brw_size", "brw_size_total", "Block read/write size in bytes"},
+			{"brw_stats", "pages_per_bulk_rw_total", pagesPerBlockRWHelp},
+			{"brw_stats", "discontiguous_pages_total", discontiguousPagesHelp},
+			{"brw_stats", "disk_io_now", diskIOsInFlightHelp},
+			{"brw_stats", "io_time_ms", ioTimeHelp},
+			{"brw_stats", "disk_io_total", diskIOSizeHelp},
+			{"degraded", "degraded_now", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded"},
+			{"filesfree", "files_free_now_total", "The number of inodes (objects) available"},
+			{"filestotal", "file_capacity", "The maximum number of inodes (objects) the filesystem can hold"},
+			{"grant_compat_disable", "grant_compat_disabled", "Binary indicator as to whether clients with OBD_CONNECT_GRANT_PARAM setting will be granted space"},
+			{"grant_precreate", "grant_precreate_capacity_bytes", "Maximum space in bytes that clients can preallocate for objects"},
+			{"job_cleanup_interval", "job_cleanup_interval_seconds", "Interval in seconds between cleanup of tuning statistics"},
+			{"job_stats", "job_read_samples_total", readSamplesHelp},
+			{"job_stats", "job_read_minimum_size_bytes", readMinimumHelp},
+			{"job_stats", "job_read_maximum_size_bytes", readMaximumHelp},
 			{"job_stats", "job_read_total", readTotalHelp},
```

Comment:
Do we want `job_read_bytes_total` here? To me, it sounds like they want just `_total` as a unit-less suffix. Since we have a unit of bytes, I believe we want to include that before the suffix. Could be I am misinterpreting the documentation though.